examples/schema/curie_map.sssom.tsv

```diff
@@ -0,0 +1,13 @@
+#curie_map:
+#  HP: http://purl.obolibrary.org/obo/HP_
+#  MP: http://purl.obolibrary.org/obo/MP_
+#  orcid: https://orcid.org/
+#mapping_set_id: https://w3id.org/sssom/commons/examples/curie_map.sssom.tsv
+#license: "https://creativecommons.org/publicdomain/zero/1.0/"
+#creator_id: orcid:0000-0002-7356-1779

```

Comment:
Please do not create examples that represents multi-valued slots as if they were singled-valued slots. `creator_id` expects a list; even if the list has one value, it should be represented as a proper list:

```
#creator_id:
#  - orcid:0000-0002-7356-1779
```

Yes, we have [explicitly allowed](https://mapping-commons.github.io/sssom/spec-formats-tsv/#multi-valued-slots-with-a-single-value) the use of the single-value syntax, but the spec is clear that this was only done for compatibility with existing sets only and that multi-valued slots should otherwise be represented as real lists. We should not provide bad examples like that.